contrib/dyn_templates/src/lib.rs

```diff
@@ -321,6 +329,18 @@ impl Template {
     /// # #[allow(unused_variables)]
     /// let template = Template::render("index", context);
     /// ```
+    ///
+    /// Using the `context` macro:
+    ///
+    /// ```rust
+    /// #[macro_use] extern crate rocket_dyn_templates;
+    /// use rocket_dyn_templates::Template;
+    ///
+    /// # #[allow(unused_variables)]
+    /// let template = Template::render("index", context!{
```

Comment:
```suggestion
    /// let template = Template::render("index", context! {
```

Can we make this style change elsewhere, too?